backend/src/main/java/com/ambacam/service/PassportService.java

```diff
@@ -0,0 +1,155 @@
+package com.ambacam.service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.ambacam.exception.ResourceBadRequestException;
+import com.ambacam.model.Autorite;
+import com.ambacam.model.Passport;
+import com.ambacam.model.Requete;
+import com.ambacam.repository.AutoriteRepository;
+import com.ambacam.repository.PassportRepository;
+import com.ambacam.repository.RequeteRepository;
+import com.ambacam.transfert.passports.Passport2PassportReadTO;
+import com.ambacam.transfert.passports.PassportCreateTO;
+import com.ambacam.transfert.passports.PassportReadTO;
+import com.ambacam.transfert.passports.PassportUpdateTO;
+
+@Service
+@Transactional(rollbackFor = Exception.class)
+public class PassportService {
+    @Autowired
+    private RequeteRepository requeteRepository;
+    
+    @Autowired
+    private AutoriteRepository autoriteRepository;
+    
+    @Autowired
+    private PassportRepository passportRepository;
+    
+    /**
+     * Create a passport
+     * 
+     * @param requeteId
+     * @param passportCreateTO
+     * 
+     * @return the new passport created
+     * 
+     * @throws ResourceNotFoundException if the requete is not found
+     * @throws ResourceBadRequestException if the autorite does not exist
+     */
+    public Passport create(Long requeteId, PassportCreateTO passportCreateTO) {
+        // find requete
+        Requete requete= findRequete(requeteId);
+        
+        // find autorite
+        Autorite delivrePar= findAutorite(passportCreateTO.getAutoriteId());
```

Comment:
Because of the collapsed string delivrePar and "=", this file seems not to have been reformatted.